packages/eslint-plugin-svelte/src/utils/ast-utils.ts

```diff
@@ -722,3 +722,16 @@ export function findVariableForReplacement(
 
 	return { hasConflict: false, variable };
 }
+
+/**
+ * Check if one node is contained withing the subtree of another node.
+ */
+export function isIn(needle: TSESTree.Node, haystack: TSESTree.Node): boolean {
+	if (needle === haystack) {
+		return true;
+	}
+	if (needle.parent === undefined || needle.parent === null) {
```

Comment:
I prefer to use `==` only in this case.

```suggestion
	if (needle.parent == null) {
```